### ISSUE TITLE
New Version with PHP 7 Support!

### ISSUE THREAD

**Issue by cosmolearning** [Widcket/dreamhost-phalcon-install#3]
Greetings,

What you've done with this script is incredible. Many DreamHost in a shared environment could benefit from this and save major server resources with the power of Phalcon. However, the current version of Phalcon supports PHP 7, which DreamHost now supports, and version 2.0.6 does not. Any chance you could compile the latest version of Phalcon so DreamHost users can benefit from the performance boosts offered by both PHP 7 and Phalcon?

Many thanks

**Comment by Widcket** [comment]
Closed with commit ad99075f23a3bd3c7ffe90e74e1bdf8aba75ad4b

**Comment by danielcosta** [comment]
I tried the latest release but without success:

[degarde]$ php -v
PHP Warning:  PHP Startup: phalcon: Unable to initialize module
Module compiled with module API=20160303
PHP    compiled with module API=20151012
These options need to match
 in Unknown on line 0
PHP 7.0.14 (cli) (built: Dec 10 2016 00:07:44) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies

**Comment by Widcket** [comment]
The latest release was compiled with php 7.1. So, it needs php 7.1.

**Comment by danielcosta** [comment]
So we need to wait Dreamhost allow us to use PHP 7.1 on its shared servers.

Thank you, anyway.

**Comment by cosmolearning** [comment]
Rita, thanks a billion for compiling Phalcon 3.1.2 for us.

Daniel, I know exactly what problem you're going through. I saw the same issue, so I decided to compile my own version of PHP in my shared hosting account. I followed this tutorial: [https://help.dreamhost.com/hc/en-us/articles/215074947-How-to-install-a-custom-version-of-PHP](url). Proceed with caution and have a backup plan. 

I got it installed and Phalcon extension to load, but I have to recompile PHP 7.1 with the other important extensions like MCrypt, OpenSSL, etc... Still working on it.
